projects/fsastorefrontlib/src/cms-components/banner/enriched-responsive-banner.component.html

```diff
@@ -1,12 +1,35 @@
 <ng-container *ngIf="component.data$ | async as data">
-  <div class="enriched-banner">
+  <div *ngIf="configStyles?.textBox" class="enriched-banner">
     <div class="image-wrapper">
       <cx-media [container]="data.media"></cx-media>
     </div>
-    <div class="enriched-banner-text">
-      <span class="enriched-banner-title">{{ data.headingText }}</span>
-      <span class="enriched-banner-details">
-        <a class="enriched-banner-styled-text" [routerLink]="data.urlLink">
+    <div
+      class="enriched-banner-text"
+      [ngStyle]="bannerStyles"
+      [ngClass]="bannerClasses"
+    >
+      <span
+        *ngIf="configStyles?.textBoxTitle"
+        [ngClass]="
+          configStyles?.textBoxType === 'box' ? 'box-title' : 'strip-title'
+        "
+        class="enriched-banner-title"
+        >{{ data.headingText }}</span
+      >
+      <span
+        *ngIf="configStyles?.textBoxDetails"
+        [ngClass]="
+          configStyles?.textBoxType === 'box'
+            ? 'enriched-banner-details-box'
+            : 'enriched-banner-details-strip'
+        "
+      >
+        <a
+          [class.enriched-banner-styled-text]="
+            configStyles?.textBoxType === 'strip'
```

Comment:
Maybe better to add 1 more property (`buttonLink: boolean` or someting like that), because with this user won't be able to use button or link on boxed model.

Reply:
^^ Guess this was not such a good idea, having in mind that this content can simply be omited in impex file.
😞

Reply:
As we agreed it'll stay as it is.